src/OrchardCore/OrchardCore.Users.Core/CommonPermissions.cs

```diff
@@ -9,21 +9,47 @@ public class CommonPermissions
         /// When authorizing request ManageUsers and pass an <see cref="IUser"/>
         /// Do not request a dynamic permission unless you are checking if the user can manage a specific role.
         /// </summary>
-        public static readonly Permission ManageUsers = new Permission("ManageUsers", "Manage Users", true);
+        public static readonly Permission ManageUsers = new("ManageUsers", "Manage security settings and all users", true);
 
         /// <summary>
         /// View users only allows listing or viewing a users profile.
         /// </summary>
-        public static readonly Permission ViewUsers = new Permission("View Users", "View Users", new[] { ManageUsers });
+        public static readonly Permission ViewUsers = new("View Users", "View user's profile", new[] { ManageUsers });
+
+        public static readonly Permission EditUsers = new(nameof(EditUsers), "Edit any user", new[] { ManageUsers }, true);
```

Comment:
nameof(EditUsers) ensure it matches the property name, but is risky if the property gets changed. Safer to use a string.

Reply:
Agree, it should be "Edit Users"